bananalyzer/data/schemas.py

```diff
@@ -163,6 +163,7 @@ def set_schema_and_goal(cls, values: Dict[str, Any]) -> Dict[str, Any]:
         from bananalyzer.data.fetch_schemas import get_fetch_schema, get_goal
 
         if values.get("schema_") is None:
+            print(f"Warning: Schema not found for Example {values['id']}")
```

Comment:
can we do a `logger.warning()`?

Reply:
we can make a separate PR for migrating all bananalyzer prints to logger